### DISCUSSION TITLE
React Native - DappShare login not working.

### DISCUSSION THREAD

**Discussion by BrandanDoesCode** [Web3Auth/Web3Auth#753] [Help]
![githubQ](https://user-images.githubusercontent.com/107240494/197501260-4b44c625-8a30-495d-9918-56bbf5c74a87.png)
"@web3auth/react-native-sdk" V 3.2.1

When attempting to log a user in with JWT and a DAPP share in order to get the users private key, the following message occurs in the web3auth popup browser:
"Unable to detect login share from the Auth Network. This may be due to slow internet connection. Check your internet speed and try again"

Is this the correct code in order to get the users private key from web3auth, using their Auth0 JWT in addition to their 24 word dapp share.

```javascript
      const web3auth = new Web3Auth(WebBrowser, {
        clientId,
        network: OPENLOGIN_NETWORK.TESTNET, // or other networks
        
        loginConfig: {
          jwt: {
            name: "NAME",
            verifier: "verifier",
            typeOfLogin: "jwt",
            clientId: "abcdefhjiqlmoeasd"
          },
        },

      });
      const info = await web3auth.login({
        loginProvider: LOGIN_PROVIDER.JWT,
        redirectUrl: uri1,
        mfaLevel: `mandatory`,
        dappShare: dappShareInput,
        curve: "secp256k1",
        extraLoginOptions: {
          domain: "domain.auth0.com",
          verifierIdField: "sub",
          id_token: currentuserAuth0Token,
        },
      });
```

The attached photo shows the error message from the remove Safari Developer debug console viewing IOS.

Regards,

**Comment by himanshuchawla009** [comment]
Hello Error shows "Duplicate token found" which means a duplicate jwt token is being passed in web3auth sdk, plz make sure to pass a unique jwt token which is not being passed before to web3auth. Also age of token shouldn't be more than 60 sec.

**Reply by irfanonk** [reply]
I am getting the same error from browser while implementing SFA. when login i am getting the Idtoken from cognito then create web3AuthProvider, when the page refresh restoring session then passing the same idtoken to conenct function and got that error.  how to persist session in the app then? or i am supposed to call connect function with another token?

**Reply by enu-kuro** [reply]
I'm implementing SFA and have the same question.
"how to persist session in the app then? or i am supposed to call connect function with another token?"
